lib.php

```diff
@@ -161,9 +168,15 @@ public function embed($urls, $name, $width, $height, $options) {
             if (isset($options['subtitles'])) {
                 $tracks = array();
                 foreach ($options['subtitles'] as $label => $subtitlefileurl) {
-                    $tracks[] = array(
-                        'file' => $subtitlefileurl->out(),
-                        'label' => $label);
+                    if ($label == 'chapters' || $label == 'thumbnails') {
+                        $tracks[] = array(
+                            'file' => $subtitlefileurl->out(),
```

Comment:
Which if statement are you referring to?

The only one there appears to check $options['subtitles'] is set?